### PR TITLE
fix(SDK-2429): fix flutter module build failure issue in existing app

### PR DESCRIPTION
**Pull request by shivamsharma2710**
Add support in CleverTap Plugin to work as a flutter module in native apps (https://docs.flutter.dev/development/add-to-app)

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -42,7 +42,7 @@ android {
 
 dependencies {
     testImplementation 'junit:junit:4.13.2'
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.3'
-    implementation 'androidx.fragment:fragment:1.3.6'
-    implementation 'androidx.core:core:1.3.0'
+    api 'com.clevertap.android:clevertap-android-sdk:4.6.3'
```
Comment (darshanclevertap) [inline android/build.gradle:45]: @shivamsharma2710 please update android sdk to v4.6.6

Comment (shivamsharma2710) [reply android/build.gradle]: This is resolved. Please check.

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -56,7 +56,7 @@ flutter {
 dependencies {
     testImplementation 'junit:junit:4.13'
     //implementation fileTree('libs')
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.3'
+    //implementation 'com.clevertap.android:clevertap-android-sdk:4.6.6'
     implementation "com.clevertap.android:push-templates:1.0.1"
```
Comment (darshanclevertap) [inline example/android/app/build.gradle:60]: @shivamsharma2710 Update push templates to latest version please?

**Review by darshanclevertap — COMMENTED** [review]
@shivamsharma2710 Added one comment

File: example/android/app/build.gradle (hunk 2 above)
Comment (shivamsharma2710) [reply example/android/app/build.gradle:60]: This is example's build.gradle and few things are below,

- The latest Push template is v1.0.7.
- We've added android 13 support in v1.0.6. 

So, should I use version prior to that(i.e. v1.0.5)?

Refer changeLog for reference -> https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTPUSHTEMPLATESCHANGELOG.md

Comment (darshanclevertap) [reply example/android/app/build.gradle:60]: Yes use 1.0.5 @shivamsharma2710

Comment (shivamsharma2710) [reply example/android/app/build.gradle]: This is done.

Comment (shivamsharma2710) [reply example/android/app/build.gradle]: Can we close this MR?

**Review by darshanclevertap — APPROVED** [review]

**Pull request merged**